packages/babel-generator/test/fixtures/sourcemaps/function-identifier-name/output.js

```diff
@@ -0,0 +1,24 @@
+function fn() {}
+
+export default function () {}
+var a = {
+  fn() {},
+
+  fn: function () {},
+  [fn]: function () {},
+  ["fn"]: function () {},
+  [function () {}]: function () {},
+  [() => {}]: function () {}
+};
+
+var x = function fn() {};
+
+var y = function fn() {};
+
+() => {};
+
+x => {};
+
+x => {};
+
+(x, y, z) => {};
```

Comment:
These should be `var x = …` form, since we could mark the arrows with names (but only if it has a `(`).